dockershrink/cli.py

```diff
@@ -52,111 +111,69 @@ def main():
         default=".dockerignore",
         help="Path to .dockerignore (default: ./.dockerignore)",
     )
-    optimize_parser.add_argument(
-        "--package-json",
-        type=str,
-        default=None,
-        help="Path to package.json (default: ./package.json or ./src/package.json)",
-    )
-    optimize_parser.add_argument(
-        "--output-dir",
-        type=str,
-        default="dockershrink.optimized",
-        help="Directory to save optimized files (default: ./dockershrink.optimized)",
-    )
-    optimize_parser.add_argument(
-        "--openai-api-key",
-        type=str,
-        default=None,
-        help="Your OpenAI API key to enable Generative AI features (alternatively, set the OPENAI_API_KEY environment variable)",
-    )
-    optimize_parser.add_argument(
-        "-v",
-        "--verbose",
-        action="store_true",
-        help="Print complete stack trace in case of failures",
-    )
+    add_common_arguments(optimize_parser)
     optimize_parser.set_defaults(func=optimize_command)
 
-    # Parse the arguments
-    args = parser.parse_args()
+    # Generate command
+    generate_parser = subparsers.add_parser(
+        "generate",
+        help="Generate new Dockerfile for NodeJS project",
+        description="""
+        Analyzes your NodeJS project and generates:
+        - Optimized multi-stage Dockerfile
+        - .dockerignore file
+        - Build instructions
+        """,
+    )
+    add_common_arguments(generate_parser)
+    generate_parser.set_defaults(func=generate_command)
 
-    # Call the appropriate function based on the subcommand
+    args = parser.parse_args()
     args.func(args)
 
 
-def version_command(args):
+def version_command():
```

Comment:
this is failing, please change it back to `version_command(args)`
the parse supplies the `args` but version command isn't receiving it, which causes failure. so even if we're not using args, we still need to receive them.